http_stubs/templatetags/stub_tags.py

```diff
@@ -16,3 +18,15 @@ def get_absolute_url_tag(context: Dict, url: Url) -> Url:
     :returns: absolute url
     """
     return context.get('request').build_absolute_uri(url)
+
+
+@register.filter(name='str_to_list')
```

Comment:
The name is a misnomer. "String to list" kind of assumes "any string to some kind of list":
1) it is incorrect as it accepts only a string representation of python dict
2) it communicates the user of the filter wrong assumption that any kind of string is acceptable

I'm not even sure this should be solved by an ad-hoc tag with wrong name which would be used exactly once just for this purpose. It looks like a bad hack.

Reply:
I renamed to `headers_to_list`.
I didn't find another way to resolve this case. Do you have any ideas?